### PR TITLE
Update docs for v1.6.0 RC2

### PR DESCRIPTION
**Pull request by carlisia**
Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required`.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Review by ashish-amarnath — CHANGES_REQUESTED** [review]
`upgrade-to-1.6.md` references the rc-1 release, that needs to be changed to rc-2

File: site/content/docs/v1.6/upgrade-to-1.6.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ If you're not yet running at least Velero v1.5, see the following:
 
     ```bash
     Client:
-        Version: v1.6.0-rc.1
+        Version: v1.6.0-rc.2
```
Comment (ashish-amarnath) [inline site/content/docs/v1.6/upgrade-to-1.6.md:32]: `site/docs/main/upgrade-to-1.6.md` is still using the `rc.1`

**Review by ashish-amarnath — APPROVED** [review]
Thanks for making the changes!

**Review by zubron — CHANGES_REQUESTED** [review]
I was looking through the previous PR for the docs upgrade and noticed this comment: https://github.com/vmware-tanzu/velero/pull/3659#pullrequestreview-627389248.

It's correct that we don't create a new version of the docs for each patch release, but our site and scripts are set up to assume that the full version string is used for pre-release versions: https://github.com/vmware-tanzu/velero/blob/main/hack/release-tools/gen-docs.sh#L37-L39.
Using v1.6 as the version for an RC means that the header on the docs looks like it's an out of date version: https://deploy-preview-3660--velero.netlify.app/docs/v1.6/ due to [this](https://github.com/vmware-tanzu/velero/blob/main/site/layouts/docs/version-warning.html).

It's not that I feel particularly strongly about this, it's just that using the GA version string for a pre-release doesn't match what our docs and scripts include, so we should either update the docs and scripts, or use the full version string.

**Comment by ashish-amarnath** [comment]
That makes sense. Thanks for that info @zubron
I agree that we don't want our pre-release to get a GA version string in the docs.
Sorry I missed that @carlisia.

**Comment by carlisia** [comment]
Haha I didn't trust the docs. The ones I wrote. 

Closing this and opening a new one.

**Pull request closed without merging**